tools/check-enforcer/Azure.Sdk.Tools.CheckEnforcer/Handlers/CheckRunHandler.cs

```diff
@@ -38,6 +38,18 @@ protected override async Task HandleCoreAsync(HandlerContext<CheckRunEventPayloa
 
                     await EvaluatePullRequestAsync(context.Client, installationId, repositoryId, sha, cancellationToken);
                 }
+                else if (payload.CheckRun.Name.Contains("("))
```

Comment:
I release this is a hack but aren't there possible pipelines that have a "(" in their name?